ably/export_test.go

```diff
@@ -106,3 +106,21 @@ func (c *Connection) OnState(ch chan<- State, states ...StateEnum) {
 func (c *Connection) OffState(ch chan<- State, states ...StateEnum) {
 	c.offState(ch, states...)
 }
+
+func (c *Connection) RecoveryKey() string {
+	c.state.Lock()
+	defer c.state.Unlock()
+	return c.recoveryKey
```

Comment:
This isn't right. Per [RTN16b](https://docs.ably.io/client-lib-development-guide/features/#RTN16b):

> Connection#recoveryKey is an attribute composed of the connectionKey, and the latest connectionSerial received on the connection, and the current msgSerial

But this seems to be returning the `resume` option passed to the library, if any.

Reply:
addressed in 4449023b755bb04b26b63f4dab7c83ccd5dba667